src/pages/Users/Login.js

```diff
@@ -17,23 +17,19 @@ class Login extends Component {
     }
   }
   render() {
-    let { email, password } = this.state.form.user
       return (
           <div>
             <div className="loginMain" >
               <form onSubmit={this.onSubmit}>
               <input
                 type="email"
                 name="email"
-                value={email}
                 placeholder="Enter Email Here"
                 onChange={this.onChange}
               />
-              <nbsp />
               <input
                 type="password"
                 name="password"
-                value={password}
```

Comment:
same comment as above.  I'd rather see this line left in.

Reply:
"